Registry/registry.chem

```diff
@@ -1,3 +1,10 @@
+rconfig  integer gsfcrad_gocart_coupling namelist,chem   1  2
```

Comment:
@ravanah @kayeekayee @stacywalters @dudhia @weiwangncar 
Folks,
I needed to add these fields for the Goddard code to build with WRF Chem. I don't know what the fields are, if that are in or out, need nesting or I/O.